<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JeuGUI</class>
 <widget class="QWidget" name="JeuGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>708</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>JeuGUI</string>
  </property>
  <widget class="QLabel" name="nomJeu">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>JEU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRejouer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>142</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Voulez-vous rejouer ?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="rejouerButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>660</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Rejouer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retourMenuButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>660</y>
     <width>114</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour au menu</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>221</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="JvsJButton">
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Joueur vs Joueur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="JvsOButton">
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Joueur vs Ordi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OvsOButton">
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Ordi vs Ordi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="retourAccueil">
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
